aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -925,7 +925,7 @@ template<> void lapackEig<double>(char jobvl, char jobvr, int n, double *a, int
   // lapack [sd]geev wants to separate output arrays: wr and wi for the real
   // and imaginary parts
   double *wr = w;
-  double *wi = w + n;
+  double *wi = w ? w + n : nullptr;
```

Comment:
How is it valid for wi to be nullptr here??

Reply:
See PR description: `lapackEig` is called twice: once with empty wi/wr to estimate size of the workspace size (by initializing `LWORK` to -1, see [`dgeev`](https://www.netlib.org/lapack/explore-html/d9/d8e/group__double_g_eeigen_ga66e19253344358f5dee1e60502b9e96f.html) doc  and next time is called to actually perform the computation.